lib/routes/minecraft/version.ts

```diff
@@ -3,12 +3,26 @@ import got from '@/utils/got';
 import { Context } from 'hono';
 
 export const route: Route = {
-    path: '/version/:versionType?/:linkType?',
+    path: '/version/:versionType?/:linkType?/:lang?',
     categories: ['game'],
-    example: '/minecraft/version',
+    example: '/minecraft/version/all/official/en',
```

Comment:
Why change the example, does this PR break the current example?

Reply:
Partially, in manifest was declared that language of plugin it's "en", and in this PR it was implemented. Example updated with all default fields for route.